### ISSUE TITLE
Missing "languages" attributes on default export

### ISSUE THREAD

**Issue by MasGaNo** [naptha/tesseract.js#887]
**Tesseract.js version 5.0.4**

**Describe the bug**
The `languages` constant object is missing from definition despite being exported in the [index.js](https://github.com/naptha/tesseract.js/blob/master/src/index.js#L14)

**To Reproduce**
1. Install `tesseract.js`
2. Try to import `import { languages } from 'tesseract.js';`
3. See error

Please attach any input image required to replicate this behavior.
![image](https://github.com/naptha/tesseract.js/assets/5469433/b47f49e6-7acf-4be1-a7b3-568296c446b6)

**Expected behavior**
The expected behavior is to have access to `languages` in TypeScript codebase and avoid these kind of [issue](https://github.com/naptha/tesseract.js/issues/858)
Also, it will help to be more TypeSafe and to create some validators rules with Zod/Yup/Joi/... by passing this object directly as source of truth.
![image](https://github.com/naptha/tesseract.js/assets/5469433/e0e97d28-9ce2-4dbe-a51d-a70f1dddbbe4)

**Device Version:**
 - OS + Version: Windows 11
 - Browser Chrome

**Additional context**
My current workaround to fix this issue is to create a `tesseract.d.ts` file in my project and add this block:
```ts
export * from 'tesseract.js';

declare module "tesseract.js" {
  export const languages: Record<'AFR' | 'AMH' | 'ARA' | 'ASM' | 'AZE' | 'AZE_CYRL' | 'BEL' | 'BEN' | 'BOD' | 'BOS' | 'BUL' | 'CAT' | 'CEB' | 'CES' | 'CHI_SIM' | 'CHI_TRA' | 'CHR' | 'CYM' | 'DAN' | 'DEU' | 'DZO' | 'ELL' | 'ENG' | 'ENM' | 'EPO' | 'EST' | 'EUS' | 'FAS' | 'FIN' | 'FRA' | 'FRK' | 'FRM' | 'GLE' | 'GLG' | 'GRC' | 'GUJ' | 'HAT' | 'HEB' | 'HIN' | 'HRV' | 'HUN' | 'IKU' | 'IND' | 'ISL' | 'ITA' | 'ITA_OLD' | 'JAV' | 'JPN' | 'KAN' | 'KAT' | 'KAT_OLD' | 'KAZ' | 'KHM' | 'KIR' | 'KOR' | 'KUR' | 'LAO' | 'LAT' | 'LAV' | 'LIT' | 'MAL' | 'MAR' | 'MKD' | 'MLT' | 'MSA' | 'MYA' | 'NEP' | 'NLD' | 'NOR' | 'ORI' | 'PAN' | 'POL' | 'POR' | 'PUS' | 'RON' | 'RUS' | 'SAN' | 'SIN' | 'SLK' | 'SLV' | 'SPA' | 'SPA_OLD' | 'SQI' | 'SRP' | 'SRP_LATN' | 'SWA' | 'SWE' | 'SYR' | 'TAM' | 'TEL' | 'TGK' | 'TGL' | 'THA' | 'TIR' | 'TUR' | 'UIG' | 'UKR' | 'URD' | 'UZB' | 'UZB_CYRL' | 'VIE' | 'YID', string>;
}
```
But it would be better to generate the definition directly from project and by importing the JSDoc on the [languages constants](https://github.com/naptha/tesseract.js/blob/master/src/constants/languages.js#L7)

Thank you.

**Comment by Balearica** [comment]
I agree this is a good suggestion, and would reduce errors like the one you linked.  However, I believe this is a breaking change so the soonest this could be implemented is Tesseract.js v6.0.  

Making this change would break code for (1) TypeScript users specifying a custom language and (2) TypeScript  users specifying multiple languages by concatenating them with `+` (e.g. `eng+chi_sim`).  I do not believe this prevents us from ever making this change, as users with multiple languages can switch to specifying them with arrays (e.g. `['eng', 'chi_sim']`) and users with custom languages (if any exist) can add a `ts-ignore` comment.  However, this does mean such a change would need to wait for the next major release. 

I will update the documentation to remove anything referencing the concatenation method for specifying multiple languages.

**Comment by hyoretsu** [comment]
This would be great, especially since from what I've seen it's an intended export.

<img width="282" height="235" alt="Image" src="https://github.com/user-attachments/assets/23fcd335-8331-436a-9aea-3e9b3e8bf4a1" />

<img width="418" height="107" alt="Image" src="https://github.com/user-attachments/assets/ff405bfb-bdc1-4f14-80b7-02e03bcdf7b4" />
